mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java

```diff
@@ -17,9 +17,7 @@
 
 package org.apache.mahout.classifier.df.tools;
 
-import java.util.List;
-import java.util.Random;
-
+import com.google.common.collect.Lists;
```

Comment:
Desist from using Guava collections now that we r on Java 7.  This import can be dispensed with.